src/components/Indicators/LoadingErrorIndicator.tsx

```diff
@@ -25,7 +24,17 @@ const RetryText = styled.Text`
   ${({ theme }) => theme.loadingErrorIndicator.retryText.css};
 `;
 
-const LoadingErrorIndicator = ({ listType, retry = () => {} }) => {
+type Props = {
+  listType: 'channel' | 'message' | 'default';
+  error?: boolean;
+  loadNextPage?: () => void;
+  retry?: () => void;
+};
+
+const LoadingErrorIndicator: React.FC<Props> & { themePath: string } = ({
```

Comment:
seems like we'll have to do this themePath thing everywhere. maybe we want a type which does this for us and we only need to reference it everywhere instead of putting this in all the components `React.FC<T> & { themePath: string }`

Reply:
Neil said he thinks we should keep it like this for now, but might pivot in the future